docs/src/main/asciidoc/security-openid-connect-client.adoc

```diff
@@ -535,6 +671,42 @@ curl -i -X GET \
 
 In contrast with the preceding command, this command returns a `403` status code.
 
+Next, test that the programmatically created OIDC client correctly acquires and propagates the token with `RestClientWithTokenHeaderParam` both in reactive and imperative (blocking) modes:
+
+[source,bash]
+----
+curl -i -X GET \
+  http://localhost:8080/frontend/user-name-with-oidc-client-token-header-param
+----
+
+This command returns the `200` status code and the name `alice`.
```

Comment:
This is another "soft" issue.
I wonder if using "This" this way might cause confusion about whether which command is being referenced. 
A reader whose eyes land in the middle of the page (without the benefit of reading the earlier parts) might tend to assume that "this" means "the following" command.
It's also a bit odd when "the preceding" seems to refer to the command that is two commands earlier. 
Would it help to explicitly state  "the preceding command" or "the following command"?

Reply:
@rolfedh Good point, makes sense